components/homme/src/theta-l/share/prim_advance_mod.F90

```diff
@@ -1444,7 +1445,14 @@ subroutine compute_andor_apply_rhs(np1,nm1,n0,qn0,dt2,elem,hvcoord,hybrid,&
         mgrad(:,:,1,k) = (dpnh_dp_i(:,:,k)*gradphinh_i(:,:,1,k)+ &
               dpnh_dp_i(:,:,k+1)*gradphinh_i(:,:,1,k+1))/2
         mgrad(:,:,2,k) = (dpnh_dp_i(:,:,k)*gradphinh_i(:,:,2,k)+ &
-              dpnh_dp_i(:,:,k+1)*gradphinh_i(:,:,2,k+1))/2
+             dpnh_dp_i(:,:,k+1)*gradphinh_i(:,:,2,k+1))/2
+
+        if (pgrad_correction==1) then
+           T0 = TREF-.0065*TREF*Cp/g     ! = 97  
```

Comment:
The constant `0.0065` should have a `_real_kind` annotation.

Reply:
This constant occurs twice in the code. Perhaps it should be made a true constant rather than hardcoded.